bootstrap.sh

```diff
@@ -67,7 +67,7 @@ function installPackages() {
   brew install colima
 
   # Install nodejs
-  brew install nodejs@18
+  brew install node@18
```

Comment:
thank you for this update. not sure why i added `nodejs@18`

Reply:
Let's go to the truth room.